ocdaction/ocdaction/settings/production.py

```diff
@@ -9,9 +9,15 @@
 # Honor the 'X-Forwarded-Proto' header for request.is_secure()
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
-SECRET_KEY = os.environ["SECRET_KEY"]
+# Security
+SECURE_SSL_REDIRECT = True
+SESSION_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = True
+
+PREPEND_WWW = True
+ALLOWED_HOSTS = ['staging-ocdaction.herokuapp.com', 'www.ocdyouthapp.org', 'ocdyouthapp.org']
```

Comment:
```suggestion
ALLOWED_HOSTS = ['staging-ocdaction.herokuapp.com', '.ocdyouthapp.org']
```
as per https://docs.djangoproject.com/en/1.11/ref/settings/#allowed-hosts, this will cover both the production URLs.